### ISSUE TITLE
Dataverse upload window

### ISSUE THREAD

**Issue by HTsybenko** [PASTA-ELN/pasta-eln#316]
Currently, the Dataverse Upload window looks like this, which might be a bit confusing to the users:
![Screenshot from 2024-07-08 11-50-58 (copy)](https://github.com/PASTA-ELN/pasta-eln/assets/125445347/5fdca186-0986-499c-86a5-ae27b9c06dd6)

Would it be possible to split it into the two fields and provide a short title, like this?
![Screenshot from 2024-07-08 15-05-44](https://github.com/PASTA-ELN/pasta-eln/assets/125445347/05b0013d-0bcf-47ec-8dcf-f0fa6afbbaee)

**Comment by jmurugan-fzj** [comment]
@HTsybenko Do you mean to add an explicit border for both the lists and also two labels with the above given texts?

**Comment by HTsybenko** [comment]
@jmurugan-fzj Yes, I think that something like that should look better. Also, with those new borders, there is no need for borders for each project in the top field. What do you think?

**Comment by jmurugan-fzj** [comment]
@HTsybenko I would keep the labels above the windows for both the lists instead of keeping them inside the list. To disable the list item border, I will try to see if this is possible via styling!

**Comment by SteffenBrinckmann** [comment]
Dataverse will be differently implented with an zenodo upload, hence old issues regarding the implementation are mute.
